gaurun/client.go

```diff
@@ -51,11 +53,24 @@ func InitGCMClient() error {
 
 func InitAPNSClient() error {
 	var err error
-	APNSClient, err = NewApnsClientHttp2(
-		ConfGaurun.Ios.PemCertPath,
-		ConfGaurun.Ios.PemKeyPath,
-		ConfGaurun.Ios.PemKeyPassphrase,
-	)
+	if ConfGaurun.Ios.PemCertPath != "" {
```

Comment:
This condition does not match processes below.

- https://github.com/mercari/gaurun/pull/138/files#diff-d841ec9d3d917f502ebe84ac84ff97aaR80
- https://github.com/mercari/gaurun/pull/138/files#diff-d841ec9d3d917f502ebe84ac84ff97aaR90

Would you like to unify?

Reply:
Hmm. I have no rules on how to handle it. Is it better to prepare a special variable?

Reply:
Or special method is better.

Reply:
How about adding a method like the following on `gaurun/conf.go`?

```go
func (s *SectionIos) IsTokenBasedProvider() bool {
	return s.TokenAuthKeyPath != "" && s.TokenAuthKeyID != ""
}

func (s *SectionIos) IsCertificateBasedProvider() bool {
	return s.PemCertPath != "" && s.PemKeyPath != ""
}
```

Reply:
Does `IsTokenBasedProvider()` need `s.TokenAuthTeamID` also?

Reply:
I fixed it.